theme/templates/includes/access-row.html

```diff
@@ -72,15 +72,15 @@
 
     <td class="user-actions">
         <i @click="undoAccess(user, index)"
-           v-if="user.can_undo && quotaHolder.id !== user.id"
+           v-if="user.can_undo && (user.user_type==='group' || (user.user_type === 'user' && quotaHolder.id !== user.id))"
```

Comment:
What is this addressing?  Is there a type that is not user or group?  I can't think of any others, but I also haven't looked up where the user_type is coming from.

Reply:
Perhaps I was making assumptions that I shouldn't. You're right that we only have `user_types` of "group" and "user". My thinking is was that we don't want to undo/remove access if the user themself is the quotaHolder. And I was thinking that groups can't be quotaHolders so only checked quotaholder for `user_type === 'user'`

Reply:
Makes sense to me with your explanation